web/core/src/jsMain/kotlin/org/jetbrains/compose/web/elements/Elements.kt

```diff
@@ -11,53 +11,7 @@ import org.jetbrains.compose.web.css.StyleSheetBuilderImpl
 import org.jetbrains.compose.web.internal.runtime.ComposeWebInternalApi
 import org.jetbrains.compose.web.internal.runtime.DomApplier
 import org.jetbrains.compose.web.internal.runtime.DomNodeWrapper
-import org.w3c.dom.Element
-import org.w3c.dom.HTMLAnchorElement
-import org.w3c.dom.HTMLAreaElement
-import org.w3c.dom.HTMLAudioElement
-import org.w3c.dom.HTMLBRElement
-import org.w3c.dom.HTMLButtonElement
-import org.w3c.dom.HTMLDataListElement
-import org.w3c.dom.HTMLDivElement
-import org.w3c.dom.HTMLElement
-import org.w3c.dom.HTMLEmbedElement
-import org.w3c.dom.HTMLFieldSetElement
-import org.w3c.dom.HTMLFormElement
-import org.w3c.dom.HTMLHRElement
-import org.w3c.dom.HTMLHeadingElement
-import org.w3c.dom.HTMLIFrameElement
-import org.w3c.dom.HTMLImageElement
-import org.w3c.dom.HTMLInputElement
-import org.w3c.dom.HTMLLIElement
-import org.w3c.dom.HTMLLabelElement
-import org.w3c.dom.HTMLLegendElement
-import org.w3c.dom.HTMLMapElement
-import org.w3c.dom.HTMLMeterElement
-import org.w3c.dom.HTMLOListElement
-import org.w3c.dom.HTMLObjectElement
-import org.w3c.dom.HTMLOptGroupElement
-import org.w3c.dom.HTMLOptionElement
-import org.w3c.dom.HTMLOutputElement
-import org.w3c.dom.HTMLParagraphElement
-import org.w3c.dom.HTMLParamElement
-import org.w3c.dom.HTMLPictureElement
-import org.w3c.dom.HTMLPreElement
-import org.w3c.dom.HTMLProgressElement
-import org.w3c.dom.HTMLSelectElement
-import org.w3c.dom.HTMLSourceElement
-import org.w3c.dom.HTMLSpanElement
-import org.w3c.dom.HTMLStyleElement
-import org.w3c.dom.HTMLTableCaptionElement
-import org.w3c.dom.HTMLTableCellElement
-import org.w3c.dom.HTMLTableColElement
-import org.w3c.dom.HTMLTableElement
-import org.w3c.dom.HTMLTableRowElement
-import org.w3c.dom.HTMLTableSectionElement
-import org.w3c.dom.HTMLTextAreaElement
-import org.w3c.dom.HTMLTrackElement
-import org.w3c.dom.HTMLUListElement
-import org.w3c.dom.HTMLVideoElement
-import org.w3c.dom.Text
+import org.w3c.dom.*
```

Comment:
Can we roll this back? We don't hive a strict policy on star imports but we do have a sort of monotonic one - that is - whenever there were list of imports we don't change them to star imports.

Reply:
Sure, but interesting rule :D